### PR TITLE
Cait get order detail

### PR DESCRIPTION
**Pull request by cait-rocco**
I made the following changes:
    created the get all orders and get one order functionality in the model, controller and route files so developers have access to see and post to the orders table. 
    updated the order model to join the products table to single order get in order to show products on 

Steps to test:
go to http://localhost:8080/bangazonAPI/v1/orders to see if a list of orders appear
go to http://localhost:8080/bangazonAPI/v1/orders/"id#" to see if the details for that order appear with product details.
got to postman and run a post for the orders table and then get to see that added order is there

### CODE REVIEW

**Comment by Rian501** [comment]
You might want to remind us to npm start before we go to the url (just in case we weren't running it before).

File: routes/index.js [hunk 1]
```diff
@@ -6,15 +6,16 @@ const router = Router();
...
 router.use(require('./products'));
 router.use(require('./paymentOptions'));
 
-
 //set up get for the home route with guidance based on bangazonAPI/v1/ route
 router.get('/', (req, res)=>{
     res.json({
 	//list of routes as endpoint guidance for developers
     "users":"bangazonAPI/v1/users",
+    "orders":"bangazonAPI/v1/orders"
```
Comment (Rian501) [inline routes/index.js]: You will need a comma at the end of this, or it breaks node when I try to run it.

File: controllers/ordersCtrl.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    .catch((err)=>{
+        next(err);
+    })
+}
+
+//MAKE SURE the dev sets POSTMAN to JSON not text.
+module.exports.postOrder = (req, res, next) => {
+    postOrderObj(req.body)
+    .then((data) => {
+        res.status(200);
```
Comment (Rian501) [inline controllers/ordersCtrl.js]: You will need to put a `.end('something')` after the (200) on line 28 in order to let the process finish.

**Review by Rian501 — CHANGES_REQUESTED** [review]
It works with a few tweaks! Will you do these little and then you are good I think.

**Comment by chrismellor08** [comment]
The part that i can test is good. Awaiting Emily's approval for part 2 👍

**Comment by Rian501** [comment]
I am able to fetch one order with a handful of products. And post. And get all!

**Review by Rian501 — APPROVED** [review]

**Comment by Rian501** [comment]
💯 12/10 would merge again

**Pull request merged**